ironfish/src/rpc/routes/transactions/sendTransaction.ts

```diff
@@ -92,53 +92,38 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
       )
     }
 
-    // Check whether amount and fee are valid or not
-    if (!CurrencyUtils.isValidOre(transaction.fee)) {
-      throw new ValidationError(
-        `Invalid transaction fee, ${transaction.fee}`,
-        undefined,
-        ERROR_CODES.VALIDATION,
-      )
-    }
-    let sum = BigInt(transaction.fee)
-    transaction.receives.map((receive) => {
-      if (!CurrencyUtils.isValidOre(receive.amount)) {
-        throw new ValidationError(
-          `Invalid transaction amount, ${receive.amount}`,
-          undefined,
-          ERROR_CODES.VALIDATION,
-        )
+    const receives = transaction.receives.map((receive) => {
```

Comment:
We iterate over receives three separate times; here, calculating the sum, and checking that the amounts are all positive. Could we consolidate into a single loop?